### PR TITLE
Ks 2022 11 ampel wording changes

### PR DESCRIPTION
**Pull request by Rineee**
Not sure about the translations files ..

### CODE REVIEW

File: locale/de_DE/LC_MESSAGES/djangojs.po [hunk 1]
```diff
@@ -1365,6 +1292,18 @@ msgstr ""
...
+
+#~ msgid "Submit"
+#~ msgstr "Einreichen"
+
+#~ msgid "Other"
+#~ msgstr "Sonstige"
+
+#~ msgid "New answer"
+#~ msgstr "Neue Antwort"
+
```
Comment (Rineee) [inline locale/de_DE/LC_MESSAGES/djangojs.po:1306]: Is this that bug again or are they really deleted?

Comment (fuzzylogic2000) [reply locale/de_DE/LC_MESSAGES/djangojs.po:1306]: I don't know, there were string changes, but we should also probably update a4 again. 
But let's only update the django.po files for now and we are safe and can handle everything a4 later!

**Review by fuzzylogic2000 — APPROVED** [review]
Perfect! 
As I have checked it out, I will remove the js.po files and then merge!

**Comment by Rineee** [comment]
> Right, the test! Will also fix that!

Thank you for fixing everything! :heart:

**Pull request merged**